pydantic/main.py

```diff
@@ -929,7 +930,7 @@ def parse_file(
     #     "The `from_orm` method is deprecated; set "
```

Comment:
Since you are here, can you uncomment this? 🤔

Reply:
I think it was commented by mistake? 🤔

Reply:
probably it's not commented by mistake. it could be because of mypy error

Reply:
Take a look at your commit message here https://github.com/pydantic/pydantic/pull/5646/commits/32c468c4de12d1d85926987f0ab591370a397023

Reply:
Why only on `orm_mode`? 🤔

Reply:
I will take a look later. Now, I've reverted the change.